src/investarena/platform/umarkets.js

```diff
@@ -495,21 +496,34 @@ export default class Umarkets {
   }
 
   parseNewOrder({ content }) {
+    const state = this.store.getState();
     const { amount, security, side } = content.order;
     const baseCurrency = get(this.quotesSettings, [security, 'baseCurrency'], '');
-    message.info(
-      `Market order created (${side.toLowerCase()} ${amount} ${baseCurrency} at price Market)`,
-    );
+    const sideCase = side.toLowerCase();
+
+    const marketOrderCreated = getTranslations(state).market_order_created_first;
+    const currencyValue = `${sideCase} ${amount} ${baseCurrency}`;
+    const priceMarket = getTranslations(state).market_order_created_last;
+    const notifyMessageInfo = `${marketOrderCreated} ${currencyValue} ${priceMarket}`;
+
+    message.info(notifyMessageInfo);
   }
+
   parseMarketOrderFilled({ content }) {
+    const state = this.store.getState();
     const { amount, averagePrice, security, side } = content.order;
     const baseCurrency = get(this.quotesSettings, [security, 'baseCurrency'], '');
     const termCurrency = get(this.quotesSettings, [security, 'termCurrency'], '');
     const price = PlatformHelper.exponentialToDecimal(averagePrice);
-    message.success(
-      `Market order filled (${side.toLowerCase()} ${amount} ${baseCurrency} at price ${price} ${termCurrency})`,
-      4,
-    );
+    const sideCase = side.toLowerCase();
+
+    const marketFilledCreated = getTranslations(state).market_order_filled_first;
+    const currencyValue = `${sideCase} ${amount} ${baseCurrency}`;
+    const atPriceMarket = getTranslations(state).market_order_filled_last;
```

Comment:
add new selector `getTranslationsByKey(state, key, defaultMessage)` and use it instead of `getTranslations(state)`

Reply:
and also use this selector in `src/client/post/Write/editorActions.js`